django_weasyprint/views.py

```diff
@@ -49,7 +49,7 @@ def get_font_config(self):
         """
         A FreeType font configuration to handle @font-config rules.
         """
-        return weasyprint.fonts.FontConfiguration()
+        return weasyprint.text.fonts.FontConfiguration()
```

Comment:
Alternatively, both import paths could be tried so as to retain compatibility with any weasyprint version like so
```
try:
    return weasyprint.text.fonts.FontConfiguration()
except AttributeError:
    # weasyprint<53 compatibility
    return weasyprint.fonts.FontConfiguration()
```
this would cause 0 runtime overhead to updated installations and still let future releases of django-weasyprint to retain the simple requirement on weasyprint>=43.

In this case, it would be interesting to create a testing matrix for different weasyprint versions with tox just like it is already done for the different django versions.

I think the value is worth the effort